app/moves/middleware.js

```diff
@@ -136,7 +104,8 @@ module.exports = {
     }
   },
   setMovesByDateRangeAndStatus: async (req, res, next) => {
-    const { dateRange, fromLocationId, status } = res.locals
+    const { dateRange, fromLocationId } = res.locals
```

Comment:
You can also get `locationId` from `req.params`. I think its best to use `req.params` within middleware by default, unless that value doesn't exist in the URL, like `dateRange`